torch/csrc/jit/passes/graph_fuser.cpp

```diff
@@ -532,6 +532,19 @@ struct GraphFuser {
     return group;
   }
 
+  bool isFusableDevice(Value *v) {
+    auto tensor_type = v->type()->cast<DimensionedTensorType>();
+    if (!tensor_type) {
```

Comment:
if the producer's value is not at least a DimensionedTensorType , shouldn't we just disable the fusing since shape prop should give all intermediate values at least ndims?